app/build.gradle

```diff
@@ -180,7 +180,7 @@ dependencies {
     }
     annotationProcessor "com.github.bumptech.glide:compiler:$glideVersion"
 
-    implementation 'com.github.jahirfiquitiva:FABsMenu:1.1.3'//Floating Action Buttons Menu (aka expandable FAB)
+    implementation 'me.jahirfiquitiva:FABsMenu:1.1.1'//Floating Action Buttons Menu (aka expandable FAB)
```

Comment:
Why did you downgrade from 1.1.3 to 1.1.1?

Reply:
It was a mistake. 1.1.3 had a fabSize attribute conflict. Upgraded it to 1.1.4.